packages/design-system/alert/index.tsx

```diff
@@ -32,6 +33,10 @@ export const AlertContext = createContext<AlertContextType>({
   alert: () => undefined,
   isMounted: false,
 });
+export let Alert: AlertContextType = {
```

Comment:
I think the `Alert`, `Toast`, and `Snackbar` components should get rid of `hooks` mode, otherwise always show `deps` warnings such as `useEffect`, `useCallback`, etc. 
but need to do in the next PR.